uwuno/src/main/java/com/learning/uwuno/room.java

```diff
@@ -1,27 +1,39 @@
 package com.learning.uwuno;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.learning.uwuno.cards.card;
 import com.learning.uwuno.cards.deck;
 import com.learning.uwuno.errors.errorNotFound;
 
-import java.util.ArrayList;
+import java.util.Collections;
+import java.util.LinkedList;
 import java.util.UUID;
 
 public class room {
     // Constants
     final private int MAX_HAND_SIZE = 7;
+    final private int MAX_PLAYERS = 10;
+    final private int MIN_PLAYERS = 2;
```

Comment:
I thought we were doing bots to play with?

Reply:
We can have more than 1 bot. It's part of the rule so just adding it for checks and future features too.